.github/renovate.json

```diff
@@ -0,0 +1,10 @@
+{
+    "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+    "addLabels": ["ok-to-test"],
+    "rebaseWhen": "behind-base-branch",
+    "recreateWhen": "never",
+    "reviewersFromCodeOwners": true,
+    "rollbackPrs": true,
```

Comment:
Looks like this may not be a recommended setting?
>Renovate does not create these rollback PRs by default, so this functionality needs to be opted-into. We recommend you do this selectively with packageRules and not globally.

ref: https://docs.renovatebot.com/configuration-options/#rollbackprs

Reply:
Removed this field from the file.